.github/workflows/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,17 @@
+blank_issues_enabled: false
+contact_links:
+  - name: New Idea?
+    url: https://github.com/CommunityToolkit/Labs-Windows/discussions?discussions_q=category%3AExperiments+category%3AIdeas
```

```suggestion
    url: https://github.com/CommunityToolkit/Labs-Windows/discussions?discussions_q=category%3AExperiments+category%3AIdeas+category%3AProposals
```
Adding proposals in here too